### ISSUE TITLE
(Bug report) Excalidraw cannot export properly.

### ISSUE THREAD

**Issue by dvai** [zadam/trilium#4271]
### Trilium Version

0.61.6beta

### What operating system are you using?

Windows

### What is your setup?

Local + server sync

### Operating System Version

Windows11

### Description

On the Windows client (It works fine on the browser), when exporting images or Library in Excalidraw, an error occurs: 

`The request is not allowed by the user agent or the platform in the current context.`

**Comment by rauenzi** [comment]
This is an issue with Excalidraw (well moreso electron) itself, not anything Trilium is doing. See relevant issue here https://github.com/excalidraw/excalidraw/issues/6750

It looks like the obsidian developer was also in that thread saying that they developed their own export dialog for excalidraw due to this issue. There are other potential ways Trilium could fix this such as monkey patching their save function or even the entire click handler.

Or, for an easier solution, Trilium could add `app.commandLine.appendSwitch("enable-experimental-web-platform-features");` to the electron startup to enable the permissions dialog that excalidraw is trying to use. That part of it is an electron issue https://github.com/electron/electron/issues/28422

**Comment by zadam** [comment]
@rauenzi I tried this, the save dialog opens, but the file is still not saved. Can you try opening the exported file?

**Comment by rauenzi** [comment]
Yes I can, I was able to save and open this very pretty picture 😄
![Untitled-2023-09-19-2246](https://github.com/zadam/trilium/assets/6865942/075f2cbe-cb26-45a4-97b8-1f32a1228392)

Only difference I noticed other than successful saving was that the default location to save was my documents folder and not my downloads folder.

**Comment by HilkopterBob** [comment]
Hey, are there Updates to this @zadam ?

**Comment by zadam** [comment]
The change is present in the latest beta.

But the results are mixed - it works on Mac, but not on my Linux box. There it just creates a weird shortcut to the non-existent file.

**Comment by zadam** [comment]
It doesn't seem to be caused by sandbox, since some file is being written (although it's nonsensical). But this electron feature is marked "experimental", so kinda expected.

**Comment by Lewiscowles1986** [comment]
So how I might work around it is to serve a blob, like you can on any HTML page. Would electron block the web-based download dialogue?

**Comment by eliandoran** [comment]
Exporting should work just fine on the latest versions of Trilium. Let me know if it's still an issue.
